pynicotine/search.py

```diff
@@ -429,8 +440,23 @@ def _file_search_request_distributed(self, msg):
 
     # Incoming Search Requests #
 
-    @staticmethod
-    def _create_file_info_list(results, max_results, permission_level):
+    def _append_file_info(self, file_list, fileinfo):
+
+        file_path = fileinfo[0]
+        file_path_lower = file_path.lower()
+        excluded_phrase = next((phrase for phrase in self.excluded_phrases if phrase in file_path_lower), False)
+
+        # Check if file path contains phrase excluded from the search network
+        if excluded_phrase:
+            log.add_search("Excluding file %(file)s from search response due to server-provided filter %(filter)s", {
```

Comment:
Avoid using generic ambiguous "filter" terminology, suggest...

"due to server-provided filter %(filter)s" -> `"because server disallows phrase \"%(phrase)s\""`

Search phrases are logged with surrounding double-quotation marks.